docs/website/docs/dlt-ecosystem/visualizations/understanding-the-tables.md

```diff
@@ -6,31 +6,137 @@ keywords: [understanding tables, loaded data, data structure]
 
 # Understanding the tables
 
-## Show tables and data in the destination
+In [Exploring the data](./exploring-the-data.md) you have seen the data that has been loaded into the
+database. Let's take a closer look at the tables that have been created.
 
+We start with a simple dlt pipeline:
+
+```py
+import dlt
+
+data = [
+    {'id': 1, 'name': 'Alice'},
+    {'id': 2, 'name': 'Bob'}
+]
+
+pipeline = dlt.pipeline(
+    pipeline_name='quick_start',
+    destination='duckdb',
+    dataset_name='mydata'
+)
+load_info = pipeline.run(data, table_name="users")
 ```
-dlt pipeline <pipeline name> show
+
+:::note
+
+Here we are using the [DuckDb destination](../destinations/duckdb.md), which is an in-memory database. Other database destinations
+will behave similarly and have similar concepts.
+
+:::
+
+## Schema
```

Comment:
let's name it `database schema` or `schema in destination` so it is not mixed with dlt schema